### DISCUSSION TITLE
[BUG] modifying readiness and livenees routes of a microservice created from docker image, the console is not able to detect the differences and will not apply the changes on commit

### DISCUSSION THREAD

**Discussion by Epado** [mia-platform/community#111] [Q&A] [answered]
'Unsaved local changes' will not appear when modifying liveness and readiness rooutes fromt he console

**Comment by albertotessarotto** [comment, marked as answer]
Hi, thank you for the issue! We have already prepared the fix for it.
With the next Console release, the fix will be released. Most likely, therefore, by the end of this week.
